packages/validation/src/diff/changes/Change.ts

```diff
@@ -0,0 +1,52 @@
+import { JsonContext } from "../../issues/index.js"
+
+export type ChangeType =
```

Comment:
This starts to feel like a whole separate package. Let's not do that right now, but probably soon-ish, also to aid any ∂ protocol work.

Reply:
Yes, I wat thinking the same.

Reply:
Later ...